### ISSUE TITLE
WCAG 2.1 AA Accessibility: Form comprehensive list of user journeys

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#31242]
## Description

We need to fully understand all of the user journeys that are available in the Sourcegraph application, and the teams that own them.

We currently have the [engineering ownership document](https://handbook.sourcegraph.com/departments/product-engineering/engineering/process/engineering_ownership#engineering-ownership). This is useful at a high level, but ultimately it is still difficult to understand how specific parts of the application map to these general areas.

We should:
1. Aim to build a comprehensive list of user journeys and the teams that own them, and update the handbook
2. Aim to improve the engineering ownership document to cover any areas that weren't covered before
3. Work with other teams and the [accessibility working group ](https://github.com/sourcegraph/sourcegraph/issues/31240) to ensure that we are able to fully cover every possible user journey

**Comment by umpox** [comment]
Current status: In progress in [#wc-accessibility-champions](https://sourcegraph.slack.com/archives/C038K3PR2M7)

**Comment by umpox** [comment]
Current status: Mostly complete

Current list of user journeys can be found:
- In [#wc-accessibility-champions](https://sourcegraph.slack.com/archives/C038K3PR2M7)
- On this tracking issue: https://github.com/sourcegraph/sourcegraph/issues/31475
- On this project board: https://github.com/orgs/sourcegraph/projects/238/views/5
